script/company.js

```diff
@@ -0,0 +1,138 @@
+function searchParam() {
+    let companyUrl = null;
+    companyUrl = window.location.search;
+    const urlParams = new URLSearchParams(companyUrl);
+    const symbol = urlParams.get('symbol');
+
+    return symbol;       
+};
+class Company {
+    constructor (symbol) {
+        this.companySymbol = symbol;
+        this.searchUrl = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/company/profile/${this.companySymbol}`;
+        this.CompanyProfile = this.getCompanyProfile();   
+    };  
+       
+    async getCompanyProfile(){
+
+        try {
+            this.setIsLoading(true);
+            const url = this.searchUrl
```

Comment:
If you're using this.searchUrl only here do you really need to have it also declared in the constructor?